### ISSUE TITLE
About the pretrain model

### ISSUE THREAD

**Issue by PanXiongAdam** [zhang-tao-whu/e2ec#15]
Hi, thank you very much for your work. Is the pre-trained model (eg. model_sbd.pth) you provided the training result parameters of the 149th epoch? Or the epoch with the best performance among all the training epochs? Because no relevant information was found in the model.pth for you provided.

**Comment by zhang-tao-whu** [comment]
Yes, the pre-trained weights are taken to be the 149.pth with the best performance in the five iterations of the experiment on SBD dataset. In fact the best performance is mostly found in 124.pth, which reaches 59.5 AP on SBD dataset.
